tests/app/dao/test_users_dao.py

```diff
@@ -221,8 +222,18 @@ def test_dao_archive_user(sample_user, sample_organisation, fake_uuid):
     create_permissions(sample_user, service_2, "view_activity")
     create_permissions(service_2_user, service_2, "manage_settings")
 
-    # make sample_user an org member
+    # make sample_user an org member with permissions
     sample_organisation.users = [sample_user]
+    new_permissions = [
+        OrganisationUserPermissions(user=sample_user, organisation=sample_organisation, permission=p)
+        for p in [OrganisationUserPermissionTypes.can_make_services_live.value]
+    ]
```

Comment:
Don’t think you need the `for` with just 1 item:
```suggestion
    new_permissions = [
        OrganisationUserPermissions(user=sample_user, organisation=sample_organisation, permission=OrganisationUserPermissionTypes.can_make_services_live.value)
    ]
```

Reply:
That's much nicer, so I've rebased the change in (you'll need to re-approve please)